.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,58 @@
+<!-- Everything wrote in between such markers before and after this phrase are comments, will not be displayed, and are to be replaced or can be deleted. The rest is to let untouched, except where specified, or your report will be ugly. Use "Preview" tab just above to check how things will be displayed. -->
+<!-- If you do not understand what is requested somewhere in here, please read Wiki pages and read other issues first. -->
+## Context
+### Prerequisites
+<!-- Put an "X" for each item you achieved or agree with, between [ ], like this: [X]. If you do not know, do not check. You are most probably not ready to submit a bug report if not all items are checked, or the bug report should not be submitted on this repository: if you do so anyway, you may receive no feedback. -->
+* [ ] I can reproduce the issue/bug that I report on
+* [ ] I run the latest version of the firmware from this repository
+* [ ] I am available to provide further details, and can reproduce the same initial situation where I identified the issue
+* [ ] I checked all available open sources, like Wiki, Internet search engines, and other issues, to document what I found in the better possible extent, confirm that something did not function as expected, and that a bug fix is required in this repository code
+* [ ] I checked for issues here, and did not find this one was already reported on
+* [ ] I understand this repository or issue tracker is not a support service for the hardware I bought, but a cooperative open-source project to play with it, and that nobody here is affiliated with ChameleonMini manufacturers or sellers
+### Environment
+<!-- Write your information on the right column between the | | characters. Do not delete/modify any | character. If you need to use a | character in your writing, or code, put them between back-quotes, `like this`. -->
+|Item|Your information|
+|---|---|
+|**Harware**|<!-- Version of your hardware: RevE Rebboted, RevG, Other, I don't known. If you don't know, it's probably written on it under the Chameleon picture. As a reminder, this repositoty only supports RevE Rebooted for now -->
+|**Buying source**|<!-- Put a URL to the merchant website where you  bought your ChameleonMini, or write a description of it. -->|
+|**Firmware**|<!-- Write the full version name of your firmware. Use the "VERSION" command, or look at "Settings" tab in GUI. -->|
+|**GUI**|<!-- Put a URL to file or repository source whre you found the GUI you are using to program or configure your ChameleonMini. If you do not use one and use commands only, type Terminal. As a reminder, rebootedGUI only is supposed to work with firmware in this repo. -->|
```

Comment:
Not true,   RebootedGUI supports both RevE and RevG

Reply:
My bad (English), this is not what I wanted to tell, but rather the opposite. On my way.